packages/amplication-client/.env

```diff
@@ -7,6 +7,7 @@ NX_REACT_APP_PLUGIN_VERSION_USE_LATEST = "true"
 
 ## this keys are client side keys, so they are not secret. It is mandatory for the correct functioning of the app
 NX_REACT_APP_BILLING_ENABLED="true"
+NX_REACT_APP_CONTACT_US_CALENDLY_URL="https://meetings-eu1.hubspot.com/oalaluf/meeting-with-oren-30"
```

Comment:
Watch out using client "environment variables" as this is only read locally in development mode